src/main/Config.cpp

```diff
@@ -900,6 +902,11 @@ Config::processConfig(std::shared_ptr<cpptoml::table> t)
             {
                 AUTOMATIC_MAINTENANCE_COUNT = readInt<uint32_t>(item);
             }
+            else if (item.first == "AUTOMATIC_SELF_CHECK_PERIOD")
```

Comment:
forgot to document this in example config

Reply:
Fixed.